src/v2/guide/components-slots.md

```diff
@@ -211,6 +211,8 @@ Now when we use the `<todo-list>` component, we can optionally define an alterna
 </todo-list>
 ```
 
+> Until 2.5.0, `slot-scope` was named `scope` and was limited to the `<template>` element.
```

Comment:
Instead of using 2 quotes, I'd suggest merging them into something like:

> Before 2.5.0, `slot-scope` was named `scope` and was limited to the `<template>` element. Since 2.5.0, `slot-scope` doesn't have this limitation and can instead be used on any element or component in the slot.